app/procedures/edit/controller.js

```diff
@@ -83,7 +83,7 @@ export default AbstractEditController.extend(ChargeActions, PatientSubmodule, {
         closeModalOnConfirm: false,
         confirmAction: 'deleteCharge',
         title: this.get('i18n').t('procedures.titles.deleteMedicationUsed'),
-        message: this.get('i18n').t('procedures.messages.deleteMedication'),
+        message: this.get('i18n').t('messages.delete_singular', { name: 'medication' }),
```

Comment:
This code is passing a non localized string when it should be passing in a localized string or it should use the name of the item being deleted.